department-of-veterans-affairs/caseflow
# Remove unneeded rake commands

### Description
Seeding and migrating DB is not needed because rake db:setup seeds the db and creates a brand new schema from the schema.rb file. 


### Acceptance Criteria
- [X] Demo is deployed with no issues

### Testing Plan
1. Deploy this branch to one of the demos
2. Ensure this demo works correctly and has seed data
